app/dojo.hoon

```diff
@@ -1624,6 +1624,92 @@
       (he-diff %txt ">=")
     (he-diff %tan u.p.cit)
   ::
+  ++  he-lens
+    |=  com/command:^^^^lens
+    ^+  +>
+    =+  ^-  source/dojo-source
+        =|  num/@
+        =-  ?.  ?=($send-api -.sink.com)  ::  XX  num is incorrect
+              sor
+            :-  0
+            :+  %as  `mark`(cat 3 api.sink.com '-poke')
+            :-  1
+            :+  %do
+              ^-  hoon
+              :+  %brtr  [%base %noun]
+              :^  %clls  [%rock %tas %post]
+                [%rock %$ endpoint.sink.com]
+              [%cnts [%& 6]~ ~]
+            sor
+        ^=  sor
+        |-  ^-  dojo-source
+        :-  num
+        ?-    -.source.com
+            $data        [%ex %sand %t data.source.com]
+            $dojo
+          %+  rash  command.source.com
+          (ifix [(punt gap) (punt gap)] dp-build:dp)
+        ::
+            $clay
+          :-  %ex
+          ^-  hoon
+          :+  %dtkt
+            [%base %noun]
+          :+  %clhp
+            [%rock %tas %cx]
+          %+  rash  pax.source.com
+          rood:(vang | /(scot %p our.hid)/home/(scot %da now.hid))
+        ::
+            $url         [%ur `~. url.source.com]
+            $api         !!
+            $get-api
+          :-  %ex
+          ^-  hoon
+          :+  %dtkt
+            [%like ~[%json] ~]
+          :*  %clsg
+              [%rock %tas %gx]
```

Comment:
I'm not sure whether it will work or not. It depends on whether Gall supports it as a synchronous scry operation or Ford has to convert it to an asynchronous request. If it's the latter, it will break, but Ford-turbo as-is should handle the first just fine.

Reply:
And yes, I do expect that issue to be fixed relatively soon.

Reply:
That makes sense. Not sure why I thought all gall scrys were broken.